manifests/charts/m9sweeper/values.yaml

```diff
@@ -64,7 +64,8 @@ dash:
           aliases:
             - index.docker.io
           login_required: false
-
+        - name: gcr.io
+          hostname: registry.k8s.io
```

Comment:
Should we set login-required to false by default?

Reply:
yes Jake, I missed it sorry adding it

Reply:
Thanks!